src/agent/src/process_options.cpp

```diff
@@ -82,6 +83,6 @@ void PrintHelp()
     std::cout << "     start           Start wazuh-agent daemon\n";
     std::cout << "     status          Get wazuh-agent daemon status\n";
     std::cout << "     stop            Stop wazuh-agent daemon\n";
-    std::cout << "     --register      Register agent\n";
+    std::cout << "     register      Register agent\n";
     std::cout << "     --h             This help message\n";
```

Comment:
Maybe just say that any other or not using any option will print the help message.